docs/reference/migration/migrate_8_0/snapshots.asciidoc

```diff
@@ -143,3 +143,16 @@ deprecated starting 7.10.0 and will be removed in 8.0.0.
 *Impact* +
 Use the <<repositories-metering-apis,Repositories Metering APIs>>.
 ====
+
+.The `xpack.searchable.snapshot.cache.size` node-level setting has been removed
```

Comment:
I don't think we should document removal of things that were never documented in the first place and only there for internal use.

Reply:
Yeah, I wasn't sure about this. I reverted the doc change.